ANS_JS_template.tex

```diff
@@ -79,11 +79,11 @@
+Growing concerns over the impact of fossil fuels on climate change have driven efforts to find low emissions sources of energy. In response, fluctuating renewable energy sources, such as solar and wind power, are growing to meet more of the electricity demand. However, maintaining reliable energy accessibility to the grid requires a stable, non-fluctuating source of power. Nuclear power plants provide nearly emissions-free, reliable energy to the grid \cite{IPCC}. To best reduce reliance on fossil fuels while ensuring reliable energy generation and profitability, nuclear renewable hybrid energy systems (NRHESs) focus on tightly coupling renewable generation with a nuclear power plant (NPP) by co-locating the generation sources on an industrial park. The industrial park is comprised of at least the NPP, the renewable energy source, and some form of industrial process that consumes the energy not used by the grid. In this paper, we analyze the computational modeling approaches currently being pursued for NRHES. We further investigate similarities between Nuclear Fuel Cycle Simulators (NFCS) and NRHESs to determine how NRHES development can benefit from the development of NFCSs. This paper begins by reviewing past research on NRHES to determine the necessary functionality of modeling software. After determining the necessary software capabilities for an NRHES model, we discuss the characteristics of a NFCS. Initially, this research sought to determine if a NRHES would benefit from a model developed in a NFCS. While we determine a full model in a NFCS is not desirable at this time, there are benefits from the development of NFCSs which NRHES can benefit from. The article concludes by discussing how characteristics NFCSs include which ongoing NRHES modeling efforts could benefit from including.
```

Comment:
I wouldn't include "Initially..." because people aren't going to really need to know that you started one way and went the other. You're overusing "benefit" a ton.